### PR TITLE
feat: upgrade summon-conjur to support Conjur CE

### LINKED ISSUE #191: Summon has been updated to be compatible with Conjur v5

### PR DESCRIPTION
**Pull request by doodlesbykumbi**
closes https://github.com/cyberark/conjur/issues/191, https://github.com/cyberark/summon-conjur/issues/12, https://github.com/cyberark/summon-conjur/issues/11

#### What does this pull request do?

+ Adds support for Conjur CE
+ Integrates with conjur-api-go
   + Support for APIKey or TokenFile auth
   + Uses standard environment variables

#### How should this be manually tested?

Run the tests.

#### Link to build in Jenkins (if appropriate)

https://jenkins.conjur.net/job/cyberark--summon-conjur/job/ce-upgrade/

### CODE REVIEW

File: test.sh [hunk 1]
```diff
@@ -1,12 +1,34 @@
...
+docker-compose up -d
+
+# Delay to allow time for Possum to come up
+# TODO: remove this once we have HEALTHCHECK in place
+docker-compose run --rm test ./wait_for_server.sh
+
+api_key=$(docker-compose exec -T possum rails r "print Credentials['cucumber:user:admin'].api_key")
+
+# Execute tests
+docker-compose run --rm \
```
Comment (ghost) [inline test.sh]: This is pretty messy for a `docker-compose` command. We could create a customer entrypoint to make this simpler and easier to read. Open an issue if you think so too.

**Review by ghost — APPROVED** [review]
This looks good to merge, tests are passing. 

https://jenkins.conjur.net/view/Conjur%205.x/job/cyberark--summon-conjur/job/ce-upgrade/

Kumbi can you set up a time to do a code read on this project? Invite anyone interesting in #about-go. Thanks!

File: test.sh (hunk 1 above)
Comment (doodlesbykumbi) [reply test.sh]: I think this was a great way to clean up the mess. No complaint here :)

**Comment by ghost** [comment]
Merging this PR also closes #12.

**Pull request merged**